lib/ftp/listener.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Ftp.Listener do
+    @moduledoc """
+    Module for testing event dispatcher.
+    TODO --> delete when testing is complete.
+    """
+  use GenServer
+  require Logger
+
+  def start_link(_) do
+    GenServer.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  def init(_args) do
+    for event <- Application.get_env(:ftp, :events), do: Ftp.EventDispatcher.register(event)
+    {:ok, %{}}
+  end
+
+  def handle_info({:ftp_event, event}, state) do
+    Logger.info("#{__MODULE__} Got an event: #{inspect(event)}")
+    {:noreply, state}
+  end
```

Comment:
Is this listener here as an example. It seems strange to have a process just to listen and log.

Reply:
Yeah it's just as an example.